docs/spec/v1beta2/helmcharts.md

```diff
@@ -390,6 +390,53 @@ Besides being reported in Events, the reconciliation errors are also logged by
 the controller. The Flux CLI offer commands for filtering the logs for a
 specific HelmChart, e.g. `flux logs --level=error --kind=HelmChart --name=<chart-name>`.
 
+### Improving resource consumption by enabling the cache
+
+When using a `HelmRepository` as Source for a `HelmChart`, the controller loads
+the repository index in-memory to find the latest version chart.
```

Comment:
```suggestion
the repository index in memory to find the latest version of the chart.
```
maybe? I think "in-memory" is more of an adjective, describes the type of something, like in-memory database, but in this case, we care talking about where it's loaded.